main/test/ca/uwaterloo/flix/runtime/TestBackend.scala

```diff
@@ -123,9 +123,11 @@ class TestBackend extends FunSuite {
     // A public Flix instance to expose the interop functions.
     val flix = createFlix()
 
+    // TODO: This should really use the default options as a starting point.
     private def createFlix(codegen: Boolean = false) = {
       val options = Options(
         debugger = Debugger.Disabled,
+        solver = ca.uwaterloo.flix.util.SolverOpts.Default,
```

Comment:
I feel like you could just write `SolverOpts.Default` and clean up the imports a bit? But I didn't look too closely.